src/test/java/org/carlspring/cloud/storage/s3fs/fileSystemProvider/NewAsynchronousFileChannelTestIT.java

```diff
@@ -27,16 +28,15 @@
 
 import com.github.marschall.memoryfilesystem.MemoryFileSystemBuilder;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import static org.carlspring.cloud.storage.s3fs.util.S3EndpointConstant.S3_GLOBAL_URI_IT;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@Tag("s3-integration-test")
-class NewAsynchronousFileChannelITTest
+@S3IntegrationTest
```

Comment:
It extends S3UnitTestBase, shouldn't we skip the `@S3IntegrationTest` annotation in such a case?
On the other hand, if this is an integration test we should skip extending the unit test I think

Reply:
Indeed it sounds misleading to have `S3UnitTestBase` and `@S3IntegrationTest` at the same time. :)
Unfortunately we inherited a lot of cases like this one from Upplication's fork and we'll have to clear them out in due time. Changing this behavior is out of the scope of this PR. :)